### PR TITLE
Create title with underline.

### PR DESCRIPTION
**Pull request by xubowenhaoren**
I'm not sure how to center the heading, so I used some hardcode: https://github.com/xubowenhaoren/yj.zgzg.io/blob/bowenx_title_with_underline/_sass/components/_title.scss#L17

image
image

### CODE REVIEW

File: _sass/components/_title.scss [hunk 1]
```diff
@@ -6,3 +6,28 @@
     font-size: 50px;
   }
 }
+
+h1.underline {
+  width: fit-content;
+  //margin-left: -20px;
```
Comment (tslmy) [inline _sass/components/_title.scss]:
```suggestion
```

File: _sass/components/_title.scss [hunk 2]
```diff
@@ -6,3 +6,28 @@
...
+  padding: 0;
+  font-weight: bold;
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+}
+
+.underline-col {
+  text-align: center;
+}
```
Comment (tslmy) [inline _sass/components/_title.scss]: https://getbootstrap.com/docs/4.0/utilities/text/
```suggestion
```

File: _layouts/tour.html [hunk 3]
```diff
@@ -28,7 +28,9 @@ <h1>{{page.title}}</h1>
         </div>
     </div>
     <div class="row pt-5">
-        <h1>2023春节游园会活动介绍</h1>
+        <div class="underline-col">
```
Comment (tslmy) [inline _layouts/tour.html]: https://getbootstrap.com/docs/4.0/utilities/text/
```suggestion
        <div class="text-center">
```

Also, what if you combine `text-center` into `<div class="row pt-5">` (i.e., `<div class="row pt-5 text-center">`)?

**Review by tslmy — CHANGES_REQUESTED** [review]

File: _layouts/tour.html (hunk 3 above)
Comment (xubowenhaoren) [reply _layouts/tour.html]: Updated.

File: _sass/components/_title.scss (hunk 2 above)
Comment (xubowenhaoren) [reply _sass/components/_title.scss]: Updated.

File: _sass/components/_title.scss (hunk 1 above)
Comment (xubowenhaoren) [reply _sass/components/_title.scss]: Removed.

**Comment by tslmy** [comment]
考虑使用`:after` 这种 pesudo-class 来生成那个 underline，而不是要求所有写HTML的人都手动把h1周围加一层 div。原因有二：

* 由于这不是个普遍做法，很多人可能会忘了
* markdown “# 标题” 生成的 h1 周围，不会有 div

image

**Comment by tslmy** [comment]
I'll close this for now, because most headings have already implemented their own underlines.

**Pull request closed without merging**